playbooks/support/scheduling.md

```diff
@@ -107,5 +78,28 @@ To override a shift:
 
    <img width=500 src="images/opsgenie-override-2.png" />
 
-[schedule]: https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/main
-[artsy-cli]: https://github.com/artsy/artsy-cli
+## **Adding Engineers**
+
+Generally, new engineers should be added to the end of the schedule rotation associated with their time zone. To do this: 
+
+1. Navigate to the [On-Call Schedule](https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/main) and edit the engineer's respective rotation.
+
+   <img width=500 src="images/opsgenie-addengs-1.png" />
+   <img width=500 src="images/opsgenie-addengs-2.png" />
+
+2. Add the engineer to the end of the participants list. Their position in the rotation will match their position in the participants list on the rotation edit page, for example:
+
+Considerations when adding a new engineer:
+
+- You should aim to avoid pairings with two new engineers and you can place the engineer earlier in the rotation to avoid this.
+- **NOTE: If you add engineers anywhere other than the end of the rotation, the schedule of the engineers who have not completed their shifts will be modified, as the order of the participants matters. You should therefore update the affected engineers to avoid scheduling conflicts.**
+- A new hire should be given 60 days after their start date to get the necessary context before having an on-call shift.
+- Check for an unbalanced number of rotation participants in rot-a vs rot-b. If staff locations result in one rotation having significantly fewer engineers, you should prioritize a balanced rotation schedule over Eastern Time Zone coverage.
+
+### Removing  Engineers
+
+When an engineer leaves:
+
+- They will be replaced with a no-one placeholder in the schedule as part of the [off-boarding checklist](https://docs.google.com/document/d/10mmqkXnYVp0ZOmF5JwVPyxkzcyIN-rqO8prHgIi11lw/edit) and replaced with a no-one placeholder
```

Comment:
Does the "no-one" placeholder only last for one rotation, or does it stay in the schedule? I'm wondering if we also need a process around removing the "no-one"?

Reply:
Another great call-out. We definitely do as it will add to all subsequent rotations too. Will add.